### ISSUE TITLE
Segmentation Fault

### ISSUE THREAD

**Issue by SimonChen1997** [nanoporetech/megalodon#342]
Hi,

I'm currently doing 6mA methylation detection with megalodon. But there is a issue that I cannot figure out.

The error file for batch running always showed "Segmentation fault". 
I also check the log.txt file, which always showed "Could not load guppy server configuration state: 'Configurations'". 
The guppy_.log file always ended with "New client connected Client 1 anonymous_client_1 id: 54572796-59e3-4fd4-bc3d-ef59ebc96342 (connection string = 'res_dna_r941_min_modbases-all-context_v001:>timeout_interval=15000>client_name=>barcode_kits=>detect_barcodes=0>move_and_trace_enabled=1>post_out=1')."

I tried to increase the RAM to even 200G, with 28 CPU and 2 GPU. But it didn't work.
Here are my scripts:

```
megalodon /scratch/qaafi/uqzche26/fast5_pass \
--guppy-server-path /home/ont-guppy-gpu-v6.4.2/bin/guppy_basecall_server \
--outputs basecalls mappings mod_mappings mods per_read_mods \
--output-directory /home/mega_out \
--reference /home/GCF_011600945.2_ASM1160094v2_genomic.fna \
--guppy-params "-d /home/rerio/basecall_models/" \
--guppy-config res_dna_r941_min_modbases-all-context_v001.cfg \
--devices "cuda:0,1" \
--processes 1 \
--num-read-enumeration-threads 1 \
--num-extract-signal-processes 1 \
--overwrite
```

Cheers.

**Comment by fsnibs10** [comment]
Hi, 

Have you resolved the problem on "segmentation fault" ? I also got the same error information promotting "Segmentation fault (core dumped)". After checking log.txt, the program terminates by promoting "Could not load guppy server configuration state".   Are there any other methods to analyze 6mA methylation data?  Looking forward to your reply. Thanks.

**Comment by SimonChen1997** [comment]
> 

No, I couldn't fix it. I think megalodon has been deprecated by Nanopore, from the previous comment.

https://github.com/nanoporetech/megalodon/issues/336#issuecomment-1428861155

Cheers

**Comment by SimonChen1997** [comment]
> Hi,
> 
> Have you resolved the problem on "segmentation fault" ? I also got the same error information promotting "Segmentation fault (core dumped)". After checking log.txt, the program terminates by promoting "Could not load guppy server configuration state". Are there any other methods to analyze 6mA methylation data? Looking forward to your reply. Thanks.

No, I couldn't fix it. I think megalodon has been deprecated by Nanopore, from the previous comment.

https://github.com/nanoporetech/megalodon/issues/336#issuecomment-1428861155

Cheers
